otter/convergence/model.py

```diff
@@ -124,7 +124,7 @@ class NovaServer(object):
 
 
 @attributes(['server_config', 'capacity',
-             Attribute('desired_lbs', default_factory=dict, instance_of=dict),
+             Attribute('desired_lbs', default_factory=dict),
```

Comment:
Is there a reason this isn't `default_factory=pset, instance_of=PSet`? Total nitpick, feel free to ignore.